examples/yaml-to-anydata-with-projection/yaml_to_anydata_with_projection.md

```diff
@@ -0,0 +1,14 @@
+# YAML to anydata conversion with projection
+
+The `data.yaml` library provides multiple APIs to selectively convert elements and attributes from a YAML source, 
```

Comment:
Multiple APIs?

Reply:
yes, parseString, parseBytes and parseStream